modules/migration-installing-migration-operator-manually.adoc

```diff
@@ -14,57 +14,49 @@ You can install the CAM Operator manually on an {product-title} 3 source cluster
 .Prerequisites
 
 * You must have `podman` installed.
+* You must have access to the link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry].
 
 .Procedure
 
+. Log in to the Red Hat Container Registry with your Customer Portal credentials:
++
+----
+# podman login registry.redhat.io
+----
+
 . Download the `operator.yml` file:
 +
 ----
 $ podman cp $(podman create registry.redhat.io/rhcam-1-0/openshift-migration-rhel7-operator:v1.0 ):/operator.yml ./
 ----
 
 . Download the `controller-3.yml` file:
-//
-// * {product-title} 3:
 +
 ----
 podman cp $(podman create registry.redhat.io/rhcam-1-0/openshift-migration-rhel7-operator:v1.0 ):/controller-3.yml ./
```

Comment:
I know this was already here, but do you mind adding the $ prompt here too please?